documentation/New-Server-Instance.md

```diff
@@ -0,0 +1,28 @@
+# _Conforma_ New server setup on Ubuntu 10.04
+
+Steps to create new server:
+
+- Have new server running on Openstack
+- Open ports 50000 and 50001 for public
+- Add the BW entry "<Country> Conforma Server" and with the public address to be used for Conforma app - to be edited by the end of this setup (here only for referênce)
+- After provided with public key, access via terminal using ssh
+- Installed Docker engine as instructed here: https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-20-04
+- Logged in to docker hub (terminal `docker login`) as msupplyfoundation user
+- Pulled last image from docker-hub:
+  sudo docker pull msupplyfoundation/conforma-demo:build-v0.2.0-1_2022-02-23_pg-12_node-14"
+- Set all required ENV VARIABLES
+- Installed Docker compose:
+  `sudo apt install docker-compose`
+- Upgraded Docker compose to v2 as instructed here: https://docs.docker.com/compose/cli-command/#install-on-linux
+- Copied demo-server folder to new server with `scp`
```

Comment:
Can you make the verb tenses consistent, probably using imperative voice. i.e. "Set new server", "Do this.."

You've got a mixture of imperative and past tense in this list.